operators/go/gen_subm_operator.sh

```diff
@@ -38,20 +37,11 @@ function initialize_subm_operator() {
   sed -i "s|REPLACE_IMAGE|quay.io/submariner/submariner-operator:$version|g" deploy/operator.yaml
   cat deploy/operator.yaml
 
-  # Create a definition namespace for SubM
-  ns_file=deploy/namespace.yaml
-cat <<EOF > $ns_file
-{
-  "apiVersion": "v1",
-  "kind": "Namespace",
-  "metadata": {
-    "name": "submariner",
-    "labels": {
-      "name": "submariner"
-    }
-  }
-}
-EOF
+  # Add example SubM namespace definition
+  ns_file_src=example_subm_ns.yaml
+  ns_file_out=deploy/namespace.yaml
+  cp $ns_file_src $ns_file_out
```

Comment:
I think this would be easier to read without the variables:

    cp example_subm_ns.yaml deploy/namespace.yaml

Reply:
Done in the latest